mod-orders/schemas/composite_po_line.json

```diff
@@ -44,7 +44,7 @@
     "adjustment": {
       "description": "the adjustment record associated with this purchase order line",
       "type": "object",
-      "$ref": "mod-orders-storage/schemas/adjustment.json"
+      "$ref": "../../mod-orders-storage/schemas/adjustment.json"
```

Comment:
Have you tried this out?  There was at one point a technical reason why the composite schemas needed to live at the top level... something like RMB couldn't resolve the references...

Reply:
It works. I checked three times: for mod-invoice-storage, mod-orders-storage, mod-orders.
I think it was fixed with [RMB-203](https://issues.folio.org/browse/RMB-203)

Reply:
Sweet, good to know!